### ISSUE TITLE
Pasarlo como prop

### ISSUE THREAD

**Issue by ortizlucas** [anaggithub/proyecto-diario#4]
https://github.com/anaggithub/proyecto-diario/blob/200387b992e4855cb73e546d6b9c31a1816fc62a/src/components/dollar-rate/dollar-rate.js#L6

Ese array dollarRate pasarlo como prop desde donde utilices el componente, y en vez de hacer  la funcion renderRates, hacer directamente el mapeo de dollarRate en la linea 41 donde ejecutas la funcion

**Comment by anaggithub** [comment]
Claro! No me había dado cuenta de eso, ahora lo corregí. Apliqué la misma idea al navbar también, y de paso lo saqué fuera en su propia carpeta.
